packages/react-components/react-components/src/index.ts

```diff
@@ -471,6 +471,7 @@ export type {
   PopoverSurfaceState,
   PopoverTriggerProps,
   PopoverTriggerState,
+  PopoverTriggerChildProps,
```

Comment:
_nit:_ Keep these ordered alphabetically

```suggestion
  PopoverTriggerChildProps,
  PopoverTriggerProps,
  PopoverTriggerState,
```